examples/immudb/src/test/java/ImmuDbTest.java

```diff
@@ -0,0 +1,66 @@
+import io.codenotary.immudb4j.Entry;
+import io.codenotary.immudb4j.ImmuClient;
+import io.codenotary.immudb4j.exceptions.CorruptedDataException;
+import io.codenotary.immudb4j.exceptions.VerificationException;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Test class for the ImmuDbClient.
+ */
+public class ImmuDbTest {
+
+    // Default port for the ImmuDb server
+    private static final int IMMUDB_PORT = 3322;
+    // Default username for the ImmuDb server
+    private final String IMMUDB_USER = "immudb";
+    // Default password for the ImmuDb server
+    private final String IMMUDB_PASSWORD = "immudb";
+    // Default database name for the ImmuDb server
+    private final String IMMUDB_DATABASE = "defaultdb";
+
+    // Test container for the ImmuDb database, with the latest version of the image and exposed port
+    @ClassRule
+    public static final GenericContainer<?> immuDbContainer = new GenericContainer<>("codenotary/immudb:latest")
```

Comment:
last one... do not use latest :)

Reply:
No problem, updated the image tag to 1.3